tests/jobs/correlated_observations_scaling/unit/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import pytest
+
+
+@pytest.fixture()
+def setup_tmpdir(tmpdir):
```

Comment:
I`ve no problem with moving this to a new file, but should be a `conftest.py` file (could potentially be the root one, or a new one in this location).